src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -41,14 +51,15 @@ const NavHeader = (props) => {
     onCancelToken,
     placeholderText,
     setIsFirstTimeOpen,
-    getSearchLayout
+    getSearchLayout,
+    opacityImage
   } = props;
   const navigation = useNavigation();
   const coverPath = topicDetail?.cover_path || null;
   const [domainHeight, setDomainHeight] = React.useState(0);
   const [searchHeight, setSearchHeight] = React.useState(0);
-  const {top} = useSafeAreaInsets();
-  const topPosition = Platform.OS === 'ios' ? top : 0;
+  const insets = useSafeAreaInsets();
+  const {width: displayWidth} = useWindowDimensions();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `useSafeAreaInsets().top` to `useSafeAreaInsets()` is a good one, as it allows for more flexibility in handling safe area insets on different sides of the screen. However, please ensure that this change does not affect the layout of your components, especially on iOS devices with notch.



<!-- This is an auto-generated comment by OSS CodeRabbit -->